servers/src/mining/stratumserver.rs

```diff
@@ -501,7 +510,7 @@ impl Handler {
 		// Issue #1159 - use a serde_json Value type to avoid extra quoting
 		let job_template_value: Value = serde_json::from_str(&job_template_json).unwrap();
 		let job_request = RpcRequest {
-			id: String::from("Stratum"),
+			id: JsonId::StrId(String::from("Stratum")),
```

Comment:
To my point about an additional explicit layer - we need to remember to wrap values like this.

Reply:
One way would be to use the `#[serde(with = "secp_ser::string_or_u64")]` like grin-wallet does.
https://github.com/mimblewimble/grin-wallet/blob/master/libwallet/src/slate_versions/v3.rs#L60